omero/sysadmins/unix/ciphers

```diff
@@ -0,0 +1,5 @@
+Weaker ciphers like ADH are disabled by default in new versions of OpenSSL
```

Comment:
Ah nice. This is an include. First I went looking for this page. Don't know if we want to identify them somehow, but big :+1: for the solution.

Reply:
They could have ``.txt`` extension